recipe.yml

```diff
@@ -152,6 +152,11 @@ stages:
         - apt autoremove -y
         - apt clean
 
+    - name: add-manpath-config
+      type: shell
+      commands: 
+        - mv /etc/man-path.config /etc/manpath.config
```

Comment:
why isnt the file in includes.container the right name?

Reply:
This is intentionally done as specified in the PR description (checkout https://github.com/Vanilla-OS/vso-image/pull/31) to prevent man-db step from failing since it can't access the system so this file is renamed to the correct name in last step to prevent the config from being used in the previous apt steps.

Locally, this isn't an issue as apt has access to the installations directories.